addons/portal_rating/static/src/chatter/frontend/message_patch.js

```diff
@@ -34,6 +34,7 @@ patch(Message.prototype, {
     },
 
     exitEditCommentMode() {
+        this.props.message.composer.clear();
```

Comment:
Sorry to add salt to the wound, but that's a bug from `message.composer` that means editing but here it's misused as reply.

Code above assigns `composer.text` when `onClikEditComment`, so I assume the composer recovery part from local storage is overridding this value?

Anyway, ok as a fix for this version, hopefully this is eventually changed to not step on bugs after bugs because of this misuse.